src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -378,7 +378,6 @@ public partial class SocketException : System.Exception
     {
         public SocketException() { }
         public SocketException(int errorCode) { }
-        public override string Message { get { return default(string); } }
```

Comment:
The most recent STABLE contract (shipped on NuGet.org) for System.Net.Primitives is 4.0.10.  In that contract, we have this for SocketException.

```c#
public class SocketException : Exception
{
    public override string Message
    {
        get
        {
            return null;
        }
    }
    public SocketError SocketErrorCode
    {
        get
        {
            return SocketError.Success;
        }
    }
    public SocketException()
    {
    }
    public SocketException(int errorCode)
    {
    }
}
```

So, you can not remove that override.